diff -urN a/configure.ac b/configure.ac
--- a/configure.ac	2023-04-28 12:28:21.000000000 -0300
+++ b/configure.ac	2023-06-02 09:35:56.729323959 -0300
@@ -5420,6 +5420,8 @@
    callgrind/tests/Makefile
    helgrind/Makefile
    helgrind/tests/Makefile
+   instrgrind/Makefile
+   instrgrind/tests/Makefile
    drd/Makefile
    drd/scripts/download-and-build-splash2
    drd/tests/Makefile
diff -urN a/Makefile.am b/Makefile.am
--- a/Makefile.am	2023-01-07 11:48:51.000000000 -0300
+++ b/Makefile.am	2023-06-02 09:35:56.729323959 -0300
@@ -8,6 +8,7 @@
 		cachegrind \
 		callgrind \
 		helgrind \
+		instrgrind \
 		drd \
 		massif \
 		dhat \
