diff -urN a/Makefile.am b/Makefile.am
--- a/Makefile.am	2025-05-20 07:59:39.000000000 -0300
+++ b/Makefile.am	2025-08-06 15:43:55.281423523 -0300
@@ -10,6 +10,7 @@
 		cachegrind \
 		callgrind \
 		helgrind \
+		instrgrind \
 		drd \
 		massif \
 		dhat \
diff -urN a/configure.ac b/configure.ac
--- a/configure.ac	2025-05-20 07:59:50.000000000 -0300
+++ b/configure.ac	2025-08-06 15:43:55.281423523 -0300
@@ -5736,6 +5736,8 @@
    callgrind/tests/Makefile
    helgrind/Makefile
    helgrind/tests/Makefile
+   instrgrind/Makefile
+   instrgrind/tests/Makefile
    drd/Makefile
    drd/scripts/download-and-build-splash2
    drd/tests/Makefile
