diff -urN a/Makefile.am b/Makefile.am
--- a/Makefile.am	2022-04-11 11:27:12.000000000 -0300
+++ b/Makefile.am	2022-05-17 10:57:18.000000000 -0300
@@ -8,6 +8,7 @@
 		cachegrind \
 		callgrind \
 		helgrind \
+		instrgrind \
 		drd \
 		massif \
 		dhat \
diff -urN a/configure.ac b/configure.ac
--- a/configure.ac	2022-04-11 11:27:12.000000000 -0300
+++ b/configure.ac	2022-05-17 10:57:18.000000000 -0300
@@ -5309,6 +5309,8 @@
    callgrind/tests/Makefile
    helgrind/Makefile
    helgrind/tests/Makefile
+   instrgrind/Makefile
+   instrgrind/tests/Makefile
    drd/Makefile
    drd/scripts/download-and-build-splash2
    drd/tests/Makefile
