diff -urN a/Makefile.am b/Makefile.am
--- a/Makefile.am	2024-04-26 13:47:41.000000000 -0300
+++ b/Makefile.am	2025-01-17 17:37:16.862632922 -0300
@@ -8,6 +8,7 @@
 		cachegrind \
 		callgrind \
 		helgrind \
+		instrgrind \
 		drd \
 		massif \
 		dhat \
diff -urN a/configure.ac b/configure.ac
--- a/configure.ac	2024-04-26 13:47:41.000000000 -0300
+++ b/configure.ac	2025-01-17 17:37:16.858632825 -0300
@@ -5666,6 +5666,8 @@
    callgrind/tests/Makefile
    helgrind/Makefile
    helgrind/tests/Makefile
+   instrgrind/Makefile
+   instrgrind/tests/Makefile
    drd/Makefile
    drd/scripts/download-and-build-splash2
    drd/tests/Makefile
