diff -urN a/Makefile.am b/Makefile.am
--- a/Makefile.am	2021-10-07 06:20:58.000000000 -0300
+++ b/Makefile.am	2023-01-03 11:39:17.000000000 -0300
@@ -8,6 +8,7 @@
 		cachegrind \
 		callgrind \
 		helgrind \
+		instrgrind \
 		drd \
 		massif \
 		dhat \
diff -urN a/configure.ac b/configure.ac
--- a/configure.ac	2022-10-24 08:07:43.000000000 -0300
+++ b/configure.ac	2023-01-03 11:39:17.000000000 -0300
@@ -5349,6 +5349,8 @@
    callgrind/tests/Makefile
    helgrind/Makefile
    helgrind/tests/Makefile
+   instrgrind/Makefile
+   instrgrind/tests/Makefile
    drd/Makefile
    drd/scripts/download-and-build-splash2
    drd/tests/Makefile
