diff --git a/Makefile.am b/Makefile.am
index 7749442..098689b 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -8,6 +8,7 @@ TOOLS = \
 		cachegrind \
 		callgrind \
 		helgrind \
+		instrgrind \
 		drd \
 		massif \
 		dhat \
diff --git a/configure.ac b/configure.ac
index b851798..d9646f7 100755
--- a/configure.ac
+++ b/configure.ac
@@ -5166,6 +5166,8 @@ AC_CONFIG_FILES([
    callgrind/tests/Makefile
    helgrind/Makefile
    helgrind/tests/Makefile
+   instrgrind/Makefile
+   instrgrind/tests/Makefile
    drd/Makefile
    drd/scripts/download-and-build-splash2
    drd/tests/Makefile
